<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMenu</class>
 <widget class="QWidget" name="MainMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainMenu</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>0</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;菜单&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QListView" name="menuListView">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>371</width>
     <height>391</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="allDishButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>500</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Academy Engraved LET</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>所有菜品信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="randomDishes">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>510</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Academy Engraved LET</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>生成随机菜品</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadAllDishesButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>500</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Dishes</string>
   </property>
  </widget>
  <widget class="QListView" name="dinnerListView">
   <property name="geometry">
    <rect>
     <x>405</x>
     <y>70</y>
     <width>371</width>
     <height>391</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;午餐&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>20</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;晚餐&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="outputToExcel">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>470</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>输出到excel</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>490</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>生成数量</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="countEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
